x/concentrated-liquidity/lp_test.go

```diff
@@ -51,10 +51,10 @@ var (
 		lowerTick:                         DefaultLowerTick,
 		upperTick:                         DefaultUpperTick,
 		currentSqrtP:                      DefaultCurrSqrtPrice,
-		amount0Desired:                    DefaultAmt0,
+		tokensProvided:                    DefaultCoins,
+		customTokensProvided:              false,
```

Comment:
This work around was the cleanest way to allow a nil sdk.Coins{} value to be provided in the test case and not get overridden.